crates/roc_std/src/roc_dict.rs

```diff
@@ -57,21 +67,3 @@ impl<K: Debug, V: Debug> Debug for RocDict<K, V> {
             .finish()
     }
 }
-
-impl<K, V> IntoIterator for RocDict<K, V> {
-    type Item = (K, V);
-    type IntoIter = roc_list::IntoIter<(K, V)>;
-
-    fn into_iter(self) -> Self::IntoIter {
-        self.0.into_iter()
-    }
-}
-
-impl<'a, K, V> IntoIterator for &'a RocDict<K, V> {
```

Comment:
As with the lists, this ought to be okay because it's `IntoIterator` for `&'a RocDict` (unlike the one for `RocDict`, above, which indeed should be deleted)